website/docs/examples/controller_square/README.md

```diff
@@ -3,6 +3,24 @@ id: controller-square
 title: Drive a Square
 ---
 
+:::info Basic Knowledge Requirements
+
+Before diving into this code, here's a quick heads-up on what you'll need to be familiar with:
+
+1. **Python Programming**: It's important to have a good grasp of Python, especially with concepts
+like `functions`, `loops`, and `classes`, since the example utilizes these fundamentals.
+2. **Asynchronous Programming with asyncio**: Familiarity with Python's asyncio for writing concurrent
+code using the `async/await` syntax.
+3. **[farm-ng Controller Service Overview](/docs/concepts/controller_service/)**:
+This overview provides a base understanding of the gRPC service the client you create will connect to.
+4. [**farm-ng Transforms & Poses Overview**](/docs/concepts/transforms_and_poses/):
+This overview provides insight into coordinate frames, transforms,
+and poses as they pertain to autonomous systems and autonomous navigation.
+5. [**farm-ng Tracks & Waypoints Overview**](/docs/concepts/controller_101/):
+This overview provides insight into compiling poses as waypoints into a Track
+that can be followed by the Amiga.
+:::
```

Comment:
FYI - @guilhermedemouraa I think it is valuable to link our relevant concepts pages for the required knowledge